forms-flow-web/src/components/CustomComponents/Pills.js

```diff
@@ -0,0 +1,14 @@
+import Badge from "react-bootstrap/Badge";
+import { CloseIcon } from "@formsflow/components";
+
+const CustomPill = ({ label, icon, bg }) => {
+  return (
+    <div>
+      <Badge pill variant={bg}>
+        {label} {icon && <CloseIcon color="#253DF4"/>}
```

Comment:
Add onclick event for CloseIcon

Reply:
Add  dataTestid = "",
  ariaLabel = "", along with label, icon, bg
  as params

Reply:
Add it in <Badge> tag